src/test/java/com/atomicjar/logging/LogFmtMarkerTest.java

```diff
@@ -0,0 +1,14 @@
+package com.atomicjar.logging;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LogFmtMarkerTest {
```

Comment:
junit 5 test classes don't need to be public

```suggestion
class LogFmtMarkerTest {
```